addons/interaction/functions/fnc_modifyTeamManagementAction.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+/*
+ * Author: veteran29
+ * Modifies the ACE_TeamManagement action to show current group color.
+ *
+ * Arguments:
+ * 0: Target <OBJECT>
+ * 1: Player <OBJECT>
+ * 2: Args <ANY>
+ * 3: Action Data <ARRAY>
+ *
+ * Return Value:
+ * None
+ *
+ *
+ * Public: No
+ */
+
+params ["_target", "", "", "_actionData"];
+
+private _color = switch (assignedTeam _target) do {
+    case "RED": {"#FF0000"};
+    case "GREEN": {"#00FF00"};
+    case "BLUE": {"#0000FF"};
+    case "YELLOW": {"#FFFF00"};
+    default {"#FFFFFF"};
```

Comment:
https://github.com/acemod/ACE3/blob/master/addons/nametags/initSettings.sqf#L12-L56

Reply:
This is not related to nametags. The assign team color icons are fixed color too. But I can use these if you want.

Reply:
https://community.bistudio.com/wiki/assignTeam

Reply:
At least use softer colors than FF0000 etc.

Reply:
Nametag ones are specifically lower contrast if I remember correctly.

I don't see an issue in using same colors for both though.

Reply:
Personally I think they should use same colors as rest of the icons. So I have two proposals:
1. I make 4 additional colored icons like in right side of my screenshot.
2. I remove colored icons for team assignment and tint them with nametag color. If no nametag component I will use fixed color.

Reply:
I would go for 2. No need to complicate this, doesn't have to be color blind safe either since there is text right next to it.